Cql/CoreTests/Elm/TestDataRequirementsAnalyzer.cs

```diff
@@ -0,0 +1,41 @@
+using System.IO;
```

Comment:
NEeds file header

Reply:
Sigh...one thing I miss about VS.....splitting a class of into another file did copy the header in VS, but not in Rider. Oh well.....

Reply:
I noticed *none* (well, except one or two) of our test files have headers, so I won't fix that here. If we want our tests with headers, we'll need to make a PR to fix it for all of them, it does not make sense to just fix up a few.

Reply:
I guess it's just easier to remove the header from the unit tests, I don't think they would be useful to anyone as-is. I'll create a ticket for that